src/modules/uv-treeviewleftpanel-module/GalleryView.ts

```diff
@@ -205,9 +205,6 @@ class GalleryView extends BaseView {
         var height = $thumb.data('height');
 
         var $img = $thumb.find('img');
-
-        $img.width(width * this.range);
```

Comment:
Not sure why these lines have been removed? Without these the images are merely cropped to size, not scaled (for me).